src/runtime/manifest.ts

```diff
@@ -861,7 +861,10 @@ ${e.message}
         if (arg.type.getEntitySchema()) {
           const fields = arg.type.getEntitySchema().fields;
           for (const name of Object.keys(fields)) {
-            fields[name].annotations = Manifest._buildAnnotationRefs(manifest, fields[name].annotations);
+            // Annotations can sometime be already processed in the case of external schemas.
+            if(fields[name].annotations && !(fields[name].annotations[0] instanceof AnnotationRef)) {
+              fields[name].annotations = Manifest._buildAnnotationRefs(manifest, fields[name].annotations);
```

Comment:
If there are annotations on a field in both - the external schema and the particle handle connection inline schema - do we override or join or else?

Reply:
What do you mean? this change does not influence that, I am only avoiding processing already processed annotations  since that will throw. "If there are annotations on a field in both": I don't know what we want in that case, since you wrote this code I was hoping you would have a story for that.